block-layer/codecs/dag-cbor.md

```diff
@@ -15,8 +15,7 @@ with the tag 42 must be a valid CID.
 ## Link Format
 
 As with all IPLD formats, DagCBOR must be able to encode [Links].
-In DagCBOR, links are encoded using the raw-binary identity [Multibase] in a
-field with a byte-string type (major type 2), with the tag 42.
+In DagCBOR, links are [CIDs] encoded using the raw-binary identity [Multibase]. That Multibase prefix (`0x00`) *must not* be omitted. They are stored as byte-string type (major type 2), with the tag 42.
```

Comment:
And it MUST be 0x00 (the identity/base256 multibase multibase).

Reply:
@Stebalien Should I rephrase it that it must be 0x00?